crates/wit-parser/src/ast/resolve.rs

```diff
@@ -759,6 +830,19 @@ impl<'a> Resolver<'a> {
         Ok(())
     }
 
+    /// For each name in the `include`, resolve the path of the include, add it to the self.includes
+    fn resolve_include(&mut self, owner: TypeOwner, i: &ast::Include<'a>) -> Result<()> {
+        let (include_from, span) = self.resolve_world_path(&i.from)?;
+        self.foreign_dep_spans.push(span);
+        self.foreign_world_spans.push(span);
```

Comment:
This seems suspicious to me reading over this because the resolved world may not be foreign?